### ISSUE TITLE
LabBuilder.py: VM creation fails due to core quote on free Azure subscription

### ISSUE THREAD

**Issue by tsluyter** [DefensiveOrigins/APT-Lab-Terraform#8]
Students who do not have a commercial / paid-for / company-provided Azure Cloud subscription have to rely on their personal free subscription. Free subscription have a quota of 4 CPU cores. Since all three VMs use 2 cores, this blocks 1/3 of the VMs from being created. 

This results in messages such as these:
```
Error: compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=<nil> Code="OperationNotAllowed" Message="Operation could not be completed as it results in exceeding approved Total Regional Cores quota. Additional details - Deployment Model: Resource Manager, Location: westeurope, Current Limit: 4, Current Usage: 4, Additional Required: 2, (Minimum) New Limit Required: 6. Submit a request for Quota increase at https://aka.ms/ProdportalCRP/?#create/Microsoft.Support/Parameters{REDACTED} by specifying parameters listed in the ‘Details’ section for deployment to succeed. Please read more about quota limits at https://docs.microsoft.com/en-us/azure/azure-supportability/regional-quota-requests."
```

**Comment by tsluyter** [comment]
I'm currently testing what happens if I change the following:

* AD: Standard_B2ms -> Standard_B1ms
* Linux: Standard_B2ms -> Standard_B2ms
* Client: Standard_B2ms -> Standard_B1ms

**Comment by tsluyter** [comment]
Can be closed when pull req #9 is applied.

**Comment by tsluyter** [comment]
Tackled by completed merge.
